account_banking_payment_export/__openerp__.py

```diff
@@ -7,7 +7,7 @@
 
 {
     'name': 'Account Banking - Payments Export Infrastructure',
-    'version': '8.0.0.3.0',
+    'version': '8.0.0.4.0',
```

Comment:
version number is strange (it should have been 8.0.1.3.0). Now it should 8.0.1.4.0